tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "target": "ES5", // ES6 === ES2015
     "lib": ["dom", "es2015", "scripthost"],
     "jsx": "react",
+    "moduleResolution": "node",
```

Comment:
What's the need / rationale for this change?

Reply:
this was an attempt to resolve an issue at shippable, thought I had removed it